widgets/use-calcite-components/README.md

```diff
@@ -0,0 +1,35 @@
+# Use calcite components widget
+
+This sample demonstrates how to use [Calcite components](https://developers.arcgis.com/calcite-design-system/components/) in custom widget.
```

Comment:
We'd better add something similar to the content in FAQ like this: before you use components in Calcite, we suggest you check components in jimu-ui...

Reply:
Where should we add these instructions?

Reply:
@mikey you can find it here: https://faq-preview.sites.afd.arcgis.com/experience-builder/guide/frequently-asked-questions/#can-i-use-components-from-calcite-when-how